packages/dashmate/src/docker/DockerCompose.js

```diff
@@ -362,21 +362,27 @@ class DockerCompose {
 
     const dockerComposeInstallLink = 'https://docs.docker.com/compose/install/';
     const dockerInstallLink = 'https://docs.docker.com/engine/install/';
+    const dockerPostInstallLinuxLink = 'https://docs.docker.com/engine/install/linux-postinstall/';
 
     // Check docker
     if (!hasbin.sync('docker')) {
       throw new Error(`Docker is not installed. Please follow instructions ${dockerInstallLink}`);
     }
 
-    const dockerVersion = await new Promise((resolve, reject) => {
-      this.docker.version((err, data) => {
-        if (err) {
-          return reject(err);
-        }
-
-        return resolve(data.Version);
+    let dockerVersion;
+    try {
+      dockerVersion = await new Promise((resolve, reject) => {
+        this.docker.version((err, data) => {
+          if (err) {
+            return reject(err);
+          }
+
+          return resolve(data.Version);
+        });
       });
-    });
+    } catch (e) {
+      throw new Error(`Docker is not accessible. Please follow instructions ${dockerPostInstallLinuxLink}`);
```

Comment:
There are multiple issues that could be there, sorted by popularity:
- docker is not started
- access issues that you are referencing too
- the wrong context is set (`docker context`)

I would display the original error and then provide 3 possible scenarios to solve.